packages/editor/src/components/parts/program/configuration/ConfigurationPart.tsx

```diff
@@ -1,4 +1,4 @@
-import type { ConfigurationData, Label, Script, Text, Widget } from '@axonivy/inscription-protocol';
+import { IVY_SCRIPT_TYPES, type ConfigurationData, type Label, type Script, type Text, type Widget } from '@axonivy/inscription-protocol';

```

Comment:
was this your editor?
because it should made a import type {}... and an addional import { IVY_S...}